cluster-essentials/go.mod

```diff
@@ -0,0 +1,121 @@
+module github.com/vmware-tanzu/tanzu-framework/cluster-essentials
+
+go 1.18
+
+replace (
+	github.com/vmware-tanzu/tanzu-framework/cluster-essentials => ./
```

Comment:
Why is this required?

Reply:
Forgot cleanup after doing some experiment.